composer.json

```diff
@@ -39,9 +39,10 @@
         "irc": "irc://irc.freenode.org/muffin"
     },
     "require": {
+        "php": ">=5.6.0",
+        "cakephp/cakephp": "~3.4"
```

Comment:
With 3.6 release round the corner we can just jump to that. Later when also changes are done we can see if there are deprecation notices if we try to lower the constraints to 3.5 or 3.4.

So please change the constraint to `3.6.0-beta2` or `dev-3.next as 3.6.0`.